src/player_hardcoded_effects.cpp

```diff
@@ -284,8 +285,36 @@ static void eff_fun_bleed( Character &u, effect &it )
             u.mod_pain( 1 );
         }
         if( one_in( 120 / intense ) ) {
+            const std::string blood_str = !u.has_trait( trait_ACIDBLOOD ) ? "Blood" : "Acid";
+            // the numerical values here coincide with the intensity thresholds at which the name of the effect changes
+            // i.e. 0-5 intensity is displayed as "Minor Bleeding", 11-20 intensity is displayed as "Bad Bleeding", etc
+            const std::map<int, std::string> intensity_strings = {
+                { 0, "%1s drips from your %2s." },
+                { 6, "%1s leaks from your %2s. " },
+                { 11, "%1s flows from your %2s." },
+                { 21, "%1s pours from your %2s!" },
+                { 31, "%1s sprays from your %2s!" }
+            };
```

Comment:
```suggestion
            static const std::map<int, translation> intensity_strings = {
                { 0, to_translation( "%1s drips from your %2s." ) },
                { 6, to_translation( "%1s leaks from your %2s. " ) },
                { 11, to_translation( "%1s flows from your %2s." ) },
                { 21, to_translation( "%1s pours from your %2s!" ) },
                { 31, to_translation( "%1s sprays from your %2s!" ) }
            };
```

Reply:
Thanks; I know basically nothing about the translation systems other than "underscores and parentheses good". :P